### ISSUE TITLE
View Own Evaluation results

### ISSUE THREAD

**Issue by dbnschools** [junpataleta/moodle-mod_threesixo#6]
Is it possible for an employee to review their own results of the evaluation without granting them access to evaluate others?  

Typically there is a team of people who are evaluating an employee.  

We'd want each student in the moodle course to be able to see and review any evaluation they might have received but not be able to evaluate others.

**Comment by junpataleta** [comment]
I am planning on creating a feature that enables teachers/admins to release feedback results to the participants.

However, the use case that you described above is not possible for this plugin at the moment. Such feedback setup seems totally different from the use case that mod_threesixo tries to address. I'll think about this and see what I can do.

**Comment by dbnschools** [comment]
I might not have been too clear in my description.  

In our scenario we'd have teacher roles evaluating student roles.  Typically these are supervisors and department heads evaluating employees.  
We do not want the student roles evaluating other students.  This is a top-down scenario so that an employee is evaluated by one or two direct supervisors.

Once a supervisor is done evaluating an employee I'd like for them to be able to view their evaluation score without access to other people's information or the ability to evaluate others.

**Comment by junpataleta** [comment]
Closing this issue. This has been implemented in 568cc98bddf2468a28f9ad7783c628d2a6e364dd and will be part of v36.0.0 and v35.3.0.

**Comment by dbnschools** [comment]
Is it now possible to only have "teacher role" do the evaluations?  No student participation?  That is what we are interested in.  Teacher roles evaluate student roles.  Students can only view their own evaluation.

**Comment by junpataleta** [comment]
Hi, apologies, but no, role-to-role feedback has not been implemented with this fix. The fix only covers the option to release feedback results to the participants. Please feel free to raise a separate issue.
